src/page/gjennomforing-detaljer/Endringsmeldinger.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { isNotStartedOrPending, isRejected, usePromise } from '../../utils/use-promise'
+import { AxiosResponse } from 'axios'
+import { Alert, Loader } from '@navikt/ds-react'
+import { EndringsmeldingerType, fetchEndringsmeldinger } from '../../api/api'
+import { Endringsmelding } from '../../component/Endringsmelding'
+
+interface EndringsmeldingerPanelProps {
+	gjennomforingId: string
+}
+
+export const Endringsmeldinger = ({ gjennomforingId }: EndringsmeldingerPanelProps) => {
+	const endringsmeldingerPromise = usePromise<AxiosResponse<EndringsmeldingerType>>(() => fetchEndringsmeldinger(gjennomforingId!))
+	const endringsmeldinger = endringsmeldingerPromise.result?.data
```

Comment:
Hvis man flytter `const endringsmeldinger = endringsmeldingerPromise.result?.data` under if-sjekkene så vil type guards sørge for at dataen er garantert å være ikke null, slik at man slipper å bruker `endringsmeldinger?.` når man skal plukke ut ting

Reply:
👍